xla/service/gpu/runtime/copy_thunk.cc

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2017 The OpenXLA Authors.
+/* Copyright 2024 The OpenXLA Authors.
```

Comment:
I think we don't update the year of existing files when something is changed. Please revert this.

Reply:
Done.